pinot-tools/src/main/java/org/apache/pinot/tools/service/api/resources/PinotServiceManagerHealthCheck.java

```diff
@@ -33,18 +35,37 @@
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import org.apache.pinot.common.utils.ServiceStatus;
+import org.apache.pinot.spi.services.ServiceRole;
+import org.apache.pinot.tools.service.PinotServiceManager;
 
 
 @Api(tags = "Health")
 @Path("/")
 public class PinotServiceManagerHealthCheck {
 
+  @Inject
+  private PinotServiceManager _pinotServiceManager;
+
   @GET
   @Produces(MediaType.TEXT_PLAIN)
   @Path("health")
   @ApiOperation(value = "Checking Pinot Service health")
   @ApiResponses(value = {@ApiResponse(code = 200, message = "Pinot Starter is healthy"), @ApiResponse(code = 503, message = "Pinot Starter is not healthy")})
   public String getStarterHealth() {
+    final Set<String> healthCheckServices = _pinotServiceManager.getHealthCheckServices();
```

Comment:
I guess you'd need to call this API somewhere but I don't see it in this PR. Is that the case?